From 9279f915aaad63f5c7538d469341deef9d784d72 Mon Sep 17 00:00:00 2001
From: Alexis Ballier <aballier@gentoo.org>
Date: Thu, 1 Mar 2012 12:05:24 -0300
Subject: [PATCH 130/145] Remove assignment to stream_copy which is never read
 after, be it by xbmc or ffmpeg. (cherry picked from
 commit 75179cb1513bd0202e1c69bafbc557f545d8f220)

---
 .../dvdplayer/DVDCodecs/Audio/DVDAudioCodecPassthroughFFmpeg.cpp     |    2 --
 1 file changed, 2 deletions(-)

diff --git a/xbmc/cores/dvdplayer/DVDCodecs/Audio/DVDAudioCodecPassthroughFFmpeg.cpp b/xbmc/cores/dvdplayer/DVDCodecs/Audio/DVDAudioCodecPassthroughFFmpeg.cpp
index 20f9aff..fbd6264 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/Audio/DVDAudioCodecPassthroughFFmpeg.cpp
+++ b/xbmc/cores/dvdplayer/DVDCodecs/Audio/DVDAudioCodecPassthroughFFmpeg.cpp
@@ -143,8 +143,6 @@ bool CDVDAudioCodecPassthroughFFmpeg::SetupMuxer(CDVDStreamInfo &hints, CStdStri
 
 
   /* set the stream's parameters */
-  muxer.m_pStream->stream_copy           = 1;
-
   m_SampleRate = hints.samplerate;
   if(!m_SampleRate && hints.codec == CODEC_ID_AC3)
     m_SampleRate = 48000;
-- 
1.7.10

